packages/utils-dom/src/htmldocument.ts

```diff
@@ -77,7 +77,7 @@ export class HTMLDocument extends Node {
      * https://developer.mozilla.org/docs/Web/API/Document/body
      */
     public get body(): HTMLElement {
-        return this.querySelectorAll('body')[0];
+        return this.querySelector('body') as HTMLElement;
```

Comment:
Do we need `as HTMLElement` or can TypeScript infer it?

Reply:
Here we break type safety because `querySelector` might return `null` but the body getter must return a `HTMLElement`.

This is the best we can do, Before TypeScript did an implicit cast from `undefined` if the array was empty because it does not support `undefined` as a default return type for arrays: https://github.com/microsoft/TypeScript/issues/13778